### PR TITLE
Fix popup element in reporting content

### LINKED ISSUE #963: The popup element in reporting content doesn't follow the mockup
## Describe the bug
The whole section has the following errors:
1. The section's title should use the `Heading 5B` style
2. The title + description section should have a 16px margin-bottom
3. Each radio button input should have an 8px distance.
4. The line-height of the final paragraph should follow the `caption` style.
5. The distance between final paragraph and the next button should be 8px.
6. The disabled state of the next button should follow [the Design Library](https://www.figma.com/file/l4Mt3dn3Ndtrvrb4aLcwXI/Design-Library?node-id=0%3A222&viewport=45%2C71%2C0.408292680978775) definition.

## Screenshots
Screen Shot 2020-05-14 at 14 04 24

### PR DESCRIPTION
**Pull request by ariessa**
## Fixes

Fixes #963 by @panchovm

## Description
+ Changed the section title to use Heading 5B style.
+ Changed the title + description section to have 16px margin-bottom.
+ Changed span element before next button to p element.
+ Changed the disabled state of the next button to follow the Design Library definition.
+ I didn't update the radio button input as it already has 8px of distance.
+ I didn't update the distance between final paragraph and the next button as it already has 8px of distance.

## Checklist
 
- [X] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [X] My pull request targets the `master` branch of the repository.
- [X] My commit messages follow [best practices][best_practices].
- [X] My code follows the established code style of the repository.
- [x] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [X] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/ContentReport/ContentReportForm.vue [hunk 1]
```diff
@@ -33,15 +33,16 @@
         </div>
       </fieldset>
 
-      <span class="caption has-text-weight-semibold has-text-grey">
+      <p class="caption has-text-weight-semibold has-text-grey margin-bottom-normal">
```
Comment (brenoferreira) [inline src/components/ContentReport/ContentReportForm.vue:36]: please revert the element to `span`

File: src/components/ContentReport/ContentReportForm.vue [hunk 2]
```diff
@@ -33,15 +33,16 @@
...
         addresses of those who complete and submit this form.
-      </span>
+      </p>
 
       <button type="button"
-              :disabled="selectedReason === null"
-              class="button next-button tiny is-info margin-top-normal is-pulled-right"
-              @click="onIssueSelected()">
+         :disabled="selectedReason === null"
+         :class="['button', 'next-button', 'tiny', 'is-info', 'is-pulled-right',
```
Comment (brenoferreira) [inline src/components/ContentReport/ContentReportForm.vue:43]: please revert the syntax to just plain string. The indentation is also wrong

File: src/components/ContentReport/ContentReportForm.vue (hunk 1 above)
Comment (ariessa) [reply src/components/ContentReport/ContentReportForm.vue]: Done

File: src/components/ContentReport/ContentReportForm.vue (hunk 2 above)
Comment (ariessa) [reply src/components/ContentReport/ContentReportForm.vue]: Done

**Pull request merged**